recipe/meta.yaml

```diff
@@ -63,13 +33,19 @@ test:
     - pip check
   requires:
     - pip
+    - psycopg2-binary >=2.7.3.1
+    - python {{ python_min }}
 
 about:
-  home: https://github.com/open-telemetry/opentelemetry-python-contrib/tree/main/instrumentation/opentelemetry-instrumentation-psycopg2
   summary: OpenTelemetry psycopg2 instrumentation
+  home: https://github.com/open-telemetry/opentelemetry-python-contrib/tree/main/instrumentation/opentelemetry-instrumentation-psycopg2
   license: Apache-2.0
   license_file: LICENSE
 
 extra:
   recipe-maintainers:
+    - conda-forge/opentelemetry-api
+    - conda-forge/opentelemetry-sdk
+    - conda-forge/opentelemetry-dbapi
```

Comment:
Are these teams OK with this? Can you ping them? Also, maybe you'all could organize a single team for this?

Reply:
@ocefpaf 

1. Agreed, I think we can consolidate teams using conda-forge/opentelemetry-api.
2. I am on the conda-forge/opentelemetry-api team and so are @mariusvniekerk and @marcelotrevisani - I believe my approval is enough to proceed to with consolidation to use conda-forge/opentelemetry-api - Please confirm. 
3. I can start raising the PRs to start fixing the otel suite (about 50+ feedstocks) to follow this approach.